### DISCUSSION TITLE
blog/how-to-create-a-timeline-of-your-cv-in-r/

### DISCUSSION THREAD

**Discussion by giscus** [AntoineSoetewey/statsandr#94] [General]
# blog/how-to-create-a-timeline-of-your-cv-in-r/

This articles shows how to build your resume or CV in a timeline format. A timeline CV allows to present your professional experiences in a chronological order

https://statsandr.com/blog/how-to-create-a-timeline-of-your-cv-in-r/

**Comment by JimmyJvR** [comment]
Dear Antoine, thanks for your nice post with that great idea, unfortunately the plot_timeline2-function in the end won't run because :
Error in theme_lares2() : could not find function "theme_lares2"

Any ideas? Thanks in advance for your reply!

**Reply by AntoineSoetewey** [reply]
Dear Jimmy,

`theme_lares2()` comes from the `lares` package. Have you installed and loaded it correctly before using this function ?

Regards,
Antoine

**Comment by JimmyJvR** [comment]
Hi Antoine,
Nice to hear from you so quickly! And yes, Lares was installed and loaded before i ran your script. I even re-installed it, restarted R too but R-machine keeps bumping in the lasts part unto not finding the function "theme_lares2" when executing "plot time-line".
New R version, all packages are up to date, almost there, what do i miss?
Thanks!
Jimmy

**Reply by AntoineSoetewey** [reply]
Hello Jimmy,

I have the same issue on my side.

Simply:
- replace `theme_lares2()` by `theme_lares()`,
- replace `theme_lares2()` by any other theme (`theme_minimal()` as in the article for instance)
- or remove it.

Hope this helps.

Regards,
Antoine

**Comment by JimmyJvR** [comment]
Dear Antoine,
That did the trick, thank you for your time & effort!
Kind regards,
Jimmy JvR

**Reply by AntoineSoetewey** [reply]
You’re welcome!

**Comment by Chc1993** [comment]
Hi Antoine, first of all thanks for share your CV template.

I have a problem editing my acadamics. data.frame rbind doesnt run because of: Error in rbind(c("my studies", "my university",  : 
  argument is missing, with no default

any ideas? thnks

**Reply by AntoineSoetewey** [reply]
Hard to help you without seeing your code. Is the code available on GitHub?

That being said, didn’t you simply forget a closing parentheses? Is my code (without any change) working on your side?

**Comment by sleuck** [comment]
Hi Antoine, thank you so much for this fantastic script! :-)

I have some questions and would be super grateful if you could answer them, but I understand if that takes too much time - feel free to be as detailed/undetailed as you wish and as your time permits.

(1) I am not sure how to manipulate the colours - changing the names of the colours in the "colour <- c("red", "blue", "green")" command does not change the colours in the plot for me. Do I have to call this change elsewhere?

(2) Apparently, ggplot2 has changed such that it is no longer possible to call the ggsave function inside a plot/function - it has, unfortunately, become impossible for me to save the image using the script. I suppose my R skills aren't good enough to see what could be done - do you have an idea? Using ggsave at the end or the png command does not work, either.

(3) I managed to change the size of the text on the y and x axes, but I am not sure how this could be done for the labels at the right-hand side - do you know which command would do that?

Thank you so much! 
Sven

**Reply by AntoineSoetewey** [reply]
Hello @sleuck,

To give you an answer as complete as possible, I would need to try some code myself. However, I’m off without my computer until August 7th.

I’ll get back to you (in this discussion) the week of August 8.
Thanks in advance for your patience.

Regards,
Antoine

**Reply by sleuck** [reply]
Thank you for getting back to me so quickly - that sounds fabulous! Some time off the computer is a great idea :-). I look forward to your answer later!

Best wishes,
Sven

**Reply by AntoineSoetewey** [reply]
Dear Sven,

I edited the code in the article, see this new version.

(1) for the colors, edit the line `scale_colour_manual(values = c("#F8766D", "#00BA38", "#619CFF"))`

(2) to save the plot, simply run the following code after having executed the code of your plot:

```
ggsave("plot1.pdf",
  width = 12,
  height = 12,
  units = "cm",
  dpi = 300
)
```

(3) edit this line of code `theme(strip.text.y = element_text(size = 10))`

Hope this helps.

Regards,
Antoine

**Reply by sleuck** [reply]
Thank you so much, Antoine!! This is fantastic! :-) I am using this for a presentation, so I'll be sure to give you credit. Have a great August!
